doc/source/tune/_tutorials/tune-xgboost.rst

```diff
@@ -0,0 +1,510 @@
+hyperparameter configurations from this search space.
+
+The output of our training run coud look like this:
+
+.. code-block::
+   :emphasize-lines: 10
+
+    +---------------------------------+------------+-------+-------------+-------------+--------------------+-------------+----------+--------+------------------+
+    | Trial name                      | status     | loc   |         eta |   max_depth |   min_child_weight |   subsample |      acc |   iter |   total time (s) |
+    |---------------------------------+------------+-------+-------------+-------------+--------------------+-------------+----------+--------+------------------|
+    | train_breast_cancer_c817a_00000 | TERMINATED |       | 0.00334038  |           8 |                  1 |    0.640256 | 0.93007  |      1 |        0.050081  |
+    | train_breast_cancer_c817a_00001 | TERMINATED |       | 0.00285335  |           4 |                  3 |    0.951621 | 0.93007  |      1 |        0.0453899 |
+    | train_breast_cancer_c817a_00002 | TERMINATED |       | 0.0597631   |           5 |                  2 |    0.96479  | 0.986014 |      1 |        0.0503612 |
+    | train_breast_cancer_c817a_00003 | TERMINATED |       | 0.000650095 |           6 |                  2 |    0.923812 | 0.951049 |      1 |        0.0588872 |
+    | train_breast_cancer_c817a_00004 | TERMINATED |       | 0.00753275  |           1 |                  1 |    0.973499 | 0.881119 |      1 |        0.0347321 |
+    | train_breast_cancer_c817a_00005 | TERMINATED |       | 0.000411214 |           5 |                  1 |    0.672503 | 0.958042 |      1 |        0.0477931 |
+    | train_breast_cancer_c817a_00006 | TERMINATED |       | 0.0940201   |           5 |                  2 |    0.711124 | 0.972028 |      1 |        0.069901  |
+    | train_breast_cancer_c817a_00007 | TERMINATED |       | 0.0372492   |           1 |                  1 |    0.76303  | 0.895105 |      1 |        0.0496318 |
+    | train_breast_cancer_c817a_00008 | TERMINATED |       | 0.000140322 |           1 |                  2 |    0.885415 | 0.909091 |      1 |        0.045424  |
+    | train_breast_cancer_c817a_00009 | TERMINATED |       | 0.000341654 |           5 |                  3 |    0.720523 | 0.937063 |      1 |        0.0657773 |
+    +---------------------------------+------------+-------+-------------+-------------+--------------------+-------------+----------+--------+------------------+
+
+The best configuration we found used ``eta=0.0940201``, ``max_depth=5``,
+``min_child_weight=2``, ``subsample=0.711124`` and reached an accuracy of
+``0.972028``.
+
+Efficient trial scheduling
+--------------------------
+Currently, Tune samples 10 different hyperparameter configurations and trains a full
+XGBoost on all of them. In our small example, training is very fast. However,
+if training takes longer, a significant amount of computer resources is spent on trials
```

Comment:
Maybe rename this to "early stopping". 

This section is also a bit verbose; maybe we can just narrow it down to the key idea (The basic idea of this scheduler: We sample a number of hyperparameter configurations. Each of these configurations is trained for a specific number of iterations. After these iterations, only the best performing hyperparameters are retained. These
are selected according to some loss metric, usually an evaluation loss.) and a short description of the parameters.

Reply:
Done